docs/prow/obligatory-security-measures.md

```diff
@@ -22,4 +22,4 @@ Remove the person from the `kyma-prow` Google project immediately. Follow [this]
 
 ### Change Secrets
 
-Change all Secrets that were valid when the person was a project member. Follow [Prow secret management](./prow-secrets-management.md) to create a new key ring and new Secrets. Then, use [Secrets populator](./../../development/tools/cmd/secretspopulator/README.md) to change all Secrets in the Prow cluster.
+Change all Secrets that were valid when the person was a project member. Follow [Prow secret management](./prow-secrets-management.md) to create a new key ring and new Secrets. Then, [External Secrets Syncer](../../prow/workload-cluster/README.md) will change all Secrets in the Prow cluster automatically.
```

Comment:
```suggestion
Change all Secrets that were valid when the person was a project member. Follow the [Prow secret management](./prow-secrets-management.md) guidelines to create a new key ring and new Secrets. Once the key ring and new Secrets are ready, the [External Secrets Syncer](../../prow/workload-cluster/README.md) changes all Secrets in the Prow cluster automatically.
```
The link ->  [External Secrets Syncer](../../prow/workload-cluster/README.md) doesn't point the External Secrets Syncer